templates/ContentGenerator/Instructor/PGProblemEditor/revert_form.html.ep

```diff
@@ -20,16 +20,18 @@
 
 		<div class="form-check mb-2<%= $foundTempFile ? '' : ' d-none' %>" id="revert-to-tmp-container">
 			<%= radio_button 'action.revert.type' => 'revert', id => 'action_revert_type_revert_id',
-				checked => undef, class => 'form-check-input' =%>
+				checked => undef, class => 'form-check-input', $foundTempFile ? () : (disabled => undef) =%>
 			<%= label_for 'action_revert_type_revert_id', class => 'form-check-label', begin =%>
 				<%== maketext('Revert to [_1]', tag('span', dir => 'ltr', $c->shortPath($c->{editFilePath}))) =%>
 			<% end =%>
 		</div>
 
 		<div class="row align-items-center mb-2">
 			<div class="col-auto">
+				% param('action.revert.type', 'backup')
+					% if !$foundTempFile && (param('action.revert.type') // '') eq 'revert';
```

Comment:
On page load if there is no temp file, no radio button is selected. The following fixes this for my tests.

```suggestion
				% param('action.revert.type', 'backup')
					% if !$foundTempFile && (param('action.revert.type') // 'revert') eq 'revert';
```

Reply:
That is not correct.  The current code works.  Your code will make it so that if the action.revert.type paramater is undefined, then it will be set to 'backup'.

Reply:
On a fresh page load (not a submit), if there is no temp file, then action.revert.type is undefined, and thus the backup needs to be selected, otherwise nothing is selected. To test revert to delete the temp file, then click on the 'Problem N' link on the navbar to do a fresh page load. There will be nothing selected in the revert tab.

Reply:
I changed it to the original code you suggested.  That is cleaner here.